apps/openmw/mwrender/creatureanimation.hpp

```diff
@@ -24,13 +24,18 @@ namespace MWRender
     // Most creatures don't need weapons/shields, so this will save some memory.
     class CreatureWeaponAnimation : public ActorAnimation, public WeaponAnimation, public MWWorld::InventoryStoreListener
     {
+        bool mShowWeapons;
+        bool mShowCarriedLeft;
```

Comment:
I suppose it is not a right place for these variables. If you want to make them protected, move them to appropriate section.

Reply:
I want to leave them private but allow getCarriedLeftShown to work with them.

Reply:
> I want to leave them private

They are already private, so your change here should be just a no-op.

Reply:
I thought they'd be unreachable. Apparently not. Hmph, okay.